androbot/handlers.py

```diff
@@ -1,11 +1,26 @@
 from aiogram import types as aiotypes
+from aiogram.dispatcher import FSMContext
 
-from . import states, views
-from .actions import Actions
-from .config import settings
+from . import schemas, states_, views
+from .actions import Actions, start_new_test
 from .errors import UserExistsException
 from .main import bot, dp
-from .schemas import TelegramUser
+from .specialty import Specialty
+from .utils import Utils
+
+
+@dp.message_handler(commands=["add_question"], state="*")
+async def add_test_question(message):
+    """
+    Добавим новых тупых вопросов
```

Comment:
```suggestion
    Добавить новые вопросы в базу данных
```

Это функция админки? или кто угодно может это сделать?